cdhweb/pages/admin.py

```diff
@@ -7,4 +9,20 @@
 # problems
 
 admin.site.unregister(Image)
-admin.site.unregister(Document)
\ No newline at end of file
+admin.site.unregister(Document)
+
+
+class LocalUserAdmin(UserAdmin):
+    '''Customize user admin list to show last login, superuser,  & groups'''
+    list_display = UserAdmin.list_display + \
+        ('is_superuser', 'is_active', 'last_login', 'group_names')
+
+    def group_names(self, obj):
+        '''custom property to display group membership'''
+        if obj.groups.exists():
+            return ', '.join(g.name for g in obj.groups.all())
+    group_names.short_description = 'groups'
+
+
+admin.site.unregister(User)
+admin.site.register(User, LocalUserAdmin)
```

Comment:
why do this for django admin when it's available in wagtail admin?

Reply:
I copied an existing local admin from another project because I've found it valuable to have an easy way to check user status on the list view. If we can incorporate all of these fields on the wagtail admin then it can go away.

Reply:
I think all we'd need to do is port `group_names`; everything else is already represented there out-of-the-box.

Reply:
do you want me to port the differences and remove this?

Reply:
idk how easy it is to customize wagtail admin for built-in stuff like `User`, but you could try doing it the same way we do other models in `wagtail_hooks.py` and see what happens? if it doesn't work, fine to just leave this in.

Reply:
I can't find where the existing `ModelAdmin` for `User` is defined! I can add a new one, and found the `add_to_settings_menu` option, but then we have two. I think it would be nice to customize this at some point, but doesn't seem urgent that we do it now.

I did find this https://github.com/wagtail/wagtail/issues/4513 — so maybe User admin isn't a normal model admin right now.